core/modules/system/system.admin.inc

```diff
@@ -1557,10 +1557,36 @@ function system_logging_settings($form, &$form_state) {
     ),
     '#description' => t('It is recommended that sites running on production environments do not display any errors.'),
   );
+  $form['watchdog_enabled_severity_levels'] = array(
+    '#type' => 'checkboxes',
+    '#title' => t("Enabled severity levels"),
+    '#options' => array(
+      WATCHDOG_EMERGENCY => t('Emergency'),
+      WATCHDOG_ALERT => t('Alert'),
+      WATCHDOG_CRITICAL => t('Critial'),
+      WATCHDOG_ERROR => t('Error'),
+      WATCHDOG_WARNING => t('Warning'),
+      WATCHDOG_NOTICE => t('Notice'),
+      WATCHDOG_INFO => t('Info'),
+      WATCHDOG_DEBUG => t('Debug'),
+      WATCHDOG_DEPRECATED => t('Deprecated'),
+    ),
+    '#default_value' => config_get('system.core', 'watchdog_enabled_severity_levels'),
```

Comment:
We could probably shorten this variable name to `dblog_severity_levels` which would remove watchdog here too.

Reply:
This is not a new variable introduced by this PR, so I will not renaming it by now. It could be done in another PR if needed, but I personnally don't like renaming variables